samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -202,9 +210,57 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                 label={'Meeting Link'}
                 required
                 placeholder={'Enter a Teams meeting link'}
-                onChange={(_, newValue) => newValue && setCallLocator({ meetingLink: newValue })}
+                onChange={(_, newValue) => {
+                  newValue ? setCallLocator({ meetingLink: newValue }) : setCallLocator(undefined);
+                }}
               />
             )}
+            {
+              /* @conditional-compile-remove(meeting-id) */ (teamsCallChosen || teamsIdentityChosen) && (
+                <Text className={teamsItemStyle} block variant="medium">
+                  <b>Or</b>
+                </Text>
+              )
+            }
+            {
+              /* @conditional-compile-remove(meeting-id) */ (teamsCallChosen || teamsIdentityChosen) && (
+                <TextField
+                  className={teamsItemStyle}
+                  iconProps={{ iconName: 'MeetingId' }}
+                  label={'Meeting Id'}
+                  required
+                  placeholder={'Enter a meeting id'}
+                  onChange={(_, newValue) => {
+                    const passcode = callLocator && 'passcode' in callLocator ? callLocator.passcode : undefined;
+                    newValue ? setCallLocator({ meetingId: newValue, passcode: passcode }) : setCallLocator(undefined);
```

Comment:
There is a bug here, if the passcode is entered first, then if meetingId is ever cleared this will reset the internal knowledge of `passcode`.
Passcode, and MeetingId, should not be cleared by the other's field change event